app/src/components/Navigation/ShoppingCart.tsx

```diff
@@ -1,9 +1,24 @@
+// noinspection ES6PreferShortImport
```

Comment:
Is there a particular reason you removed the ES inspection for this file? Not sure I understand why.

Reply:
Cypress does not pick up the typescript config that we can use absolute paths prefixed with @.
eslint thinks we should use the shorter and better understandable absolute paths.
Until we fixed #18, it will be like this :disappointed: